wallets/trezor/meta.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "trezor",
+    "displayName": "Trezor",
+    "downloadURLs": {
+        "chrome": "https://trezor.io/learn/a/download-verify-trezor-suite",
+        "brave": "https://trezor.io/learn/a/download-verify-trezor-suite",
```

Comment:
You could remove chrome and brave options.

Reply:
fixed